.github/workflows/platformDeploy.yml

```diff
@@ -353,6 +353,15 @@ jobs:
         env:
           CF_API_KEY: ${{ secrets.CLOUDFLARE_TOKEN }}
 
+      - name: Verify that the deployed version matches what we expect
+        run: |
+          LOCAL_VERSION="$(npm run print-version --silent)"
+          DOWNLOADED_VERSION="$(wget -q -O /dev/stdout https://${{ env.SHOULD_DEPLOY_PRODUCTION != 'true' && 'staging.' || '' }}new.expensify.com | jq -r '.version')"
```

Comment:
Mentioned on slack, but this looks a little too fancy for me, why not two steps? One for production and one for staging?